Modules/Scripted/DICOMLib/DICOMUtils.py

```diff
@@ -0,0 +1,96 @@
+import os
+import vtk, qt, ctk, slicer
+import logging
+
+#########################################################
+#
+#
+comment = """
+
+  DICOMUtils is a collection of static-like DICOM
+  utility functions facilitating convenient scripted
+  use of the DICOM database
+
+"""
+#
+######################################################### 
+
+#------------------------------------------------------------------------------
+# Get patient UID by patient name for easy loading of a patient
+def patientUIDByPatientName(name):
```

Comment:
A small improvement possibility (probably not important for tests but could be useful for general use): In DICOM usually the patient name and patient ID are used together to identify a patient. It may be useful to add an optional patientID parameter that would allow to search for a specific patient name AND id.